docs/MIGRATION_V1_V2.md

```diff
@@ -0,0 +1,80 @@
+| [E053](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E053) 	| [OverlappingFrequencyNotice](/RULES.md#OverlappingFrequencyNotice)                                                                                                                                                                        	| [ERROR](/RULES.md#errors)     	|
+| [E054](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E054) 	| [BlockTripsWithOverlappingStopTimesNotice](/RULES.md#BlockTripsWithOverlappingStopTimesNotice)                                                                                                                                            	| [ERROR](/RULES.md#errors)     	|
+| [E055](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E055) 	| [FeedInfoLangAndAgencyLangMismatchNotice](/RULES.md#FeedInfoLangAndAgencyLangMismatchNotice)                                                                                                                                              	| [WARNING](/RULES.md#warnings) 	|
+| [E056](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E056) 	| [MissingCalendarAndCalendarDateFilesNotice](/RULES.md#MissingCalendarAndCalendarDateFilesNotice)                                                                                                                                          	| [ERROR](/RULES.md#errors)     	|
+| [E057](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E057) 	| [DecreasingOrEqualStopTimeDistanceNotice](/RULES.md#DecreasingOrEqualStopTimeDistanceNotice)                                                                                                                                              	| [ERROR](/RULES.md#errors)     	|
+| [E058](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E058) 	| [DecreasingOrEqualShapeDistanceNotice](/RULES.md#DecreasingOrEqualShapeDistanceNotice)                                                                                                                                                    	| [ERROR](/RULES.md#errors)     	|
+| [E059](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E059) 	| Not implemented (new architecture)                                                                                                                                                                                                        	| N/A                              	|
+| [E060](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E060) 	| Not implemented (new architecture)                                                                                                                                                                                                        	| N/A                              	|
```

Comment:
E060 and E061 are actually covered under `SystemErrors`, right?

Reply:
Yes, modifying this. Thanks.